docs/faq.md

```diff
@@ -0,0 +1,157 @@
+# Feldera Frequently Asked Questions
+
+## What is Feldera
+
+Feldera is an early-stage start-up building a new data processing
+engine for *streaming data*.  In the rest of this document we will
+write "F" for Feldera.
+
+Using indistry jargon, the F engine can be described in several ways:
+
+- a continuous query engine
+- a standing query engine
+- performing incremental view maintenance
+- implementing materialized views
+
+## What is streaming data?
+
+Data that is being continuously modified, usually at a high rate
+(using insertions, deletions, and updates).  F is designed for
+database-like data: numbers, strings, dates, JSON.
+
+## How about streaming video?
+
+F is not designed for media processing.
+
+## What are the "killer apps" for F?
```

Comment:
This section doesn't really describe any specific apps.  I think this is fair enough, we will hopefully add some shortly.

It is also focused on performance and cost. I would expand the "complex analysis" item: (1) complex SQL queries that other streaming engines don't support, (2) queries that require maintaining a lot of state that doesn't fit in memory.